docs/api/instruments/rigol/rigol_dg800.rst

```diff
@@ -0,0 +1,10 @@
+##############################
+RIGOL DG800 WAVEFORM GENERATOR
```

Comment:
No need to use all caps, as it will be rendered on the readthedocs page as a title anyways.